src/main/java/com/dataguise/saas/dto/ColumnViewUIFormatterDTO.java

```diff
@@ -0,0 +1,176 @@
+package com.dataguise.saas.dto;
+
+import com.google.gson.Gson;
+
+public class ColumnViewUIFormatterDTO {
+
+    String columnName;
+    String dataType;
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.design.VisibilityModifierCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/dto/ColumnViewUIFormatterDTO.java L8
    
    Variable &#39;dataType&#39; must be private and have accessor methods.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1456160610>*